src/CLIMAParameters.jl

```diff
@@ -1,4 +1,4 @@
-module CLIMAParameters
```

Comment:
We'll need to rename this filename, too. Also, I recommend that that is done in a separate commit, since git can sometimes have a hard time with grouping filename changes with code changes.